knowledge-base/set-the-file-path-of-json-or-csv-datasource-through-report-parameter.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Set the File Path of JSON or CSV DataSource Through Report Parameter
+description: This article explains how to set the file path of the CSV or JSON file of  JSON or CSV DataSource through a report parameter
+type: how-to
+page_title: Set the File Path of JSON or CSV DataSource Through Report Parameter
+slug: set-the-file-path-of-json-or-csv-datasource-through-report-parameter
+position: 
+tags: JsonDataSource, CSV DataSource, Report Parameter
```

Comment:
For consistency use either JsonDataSource and CsvDataSource, or JSON DataSource and CSV DataSource